connector/src/main/java/org/jboss/as/connector/logging/ConnectorLogger.java

```diff
@@ -921,4 +921,11 @@
     @Message(id = 117, value = "%s is not a valid %s implementation")
     OperationFailedException notAValidDataSourceClass(String clz, String dsClz);
 
+    @LogMessage(level = INFO)
+    @Message(id = 118, value = "Binding connection factory named %s to alias %s")
```

Comment:
I'm not sure but I think this log is also used when binding Admin Objects.

Reply:
not the same log entry